### PR TITLE
larch package

### PR DESCRIPTION
**Pull request by jpn--**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by jpn--** [comment]
Hello @conda-forge/help-python-c, I think this is ready to go.  Seems to pass all the build checks.  Thanks!

**Comment by jan-janssen** [comment]
@conda-forge/staged-recipes, ready for review

File: recipes/larch/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  number: 0
+  skip: True  # [py<37]
+  script:
+    - "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+
+  build:
+    - {{ compiler('c') }}
+    - compilers
```
Comment (scopatz) [inline recipes/larch/meta.yaml]: the `compilers` package should not be used in recipes.  It is only for use by end-users.  please use the `{{ compiler() }}`  calls

Comment (jpn--) [reply recipes/larch/meta.yaml]: Removed the “compilers” dependency.

**Comment by jpn--** [comment]
@conda-forge/staged-recipes, the comments have been resolved and this is ready for [re?]review.  Thanks!

**Pull request merged**